paddle/fluid/framework/paddle2cinn/build_cinn_pass.cc

```diff
@@ -527,9 +535,12 @@ void AnalyseClusterVariables(
   // remove output node from cluster_inputs,
   // and add cluster_internals node
   for (auto* var_node : *cluster_outputs) {
-    if (cluster_inputs->count(var_node) > 0) {
+    if ((cluster_inputs->count(var_node) > 0) ||
```

Comment:
NIT:

Can we write:
`if (cluster_inputs->count(var_node) > 0 || unused_outputs.count(var_node->Name()))` without many parentheses?